docs/linux/reporting_kernel_bugs.drawio

```diff
@@ -0,0 +1,287 @@
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-55" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;exitX=0.5;exitY=1;exitDx=0;exitDy=0;entryX=0.5;entryY=0;entryDx=0;entryDy=0;" edge="1" parent="1" source="oGheDNism3NP_6yb4b1f-14" target="oGheDNism3NP_6yb4b1f-54">
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-14" value="Is fix publicly released?" style="rhombus;whiteSpace=wrap;html=1;" vertex="1" parent="1">
+          <mxGeometry x="380" y="225" width="100" height="90" as="geometry" />
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-18" value="" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;exitX=0.5;exitY=1;exitDx=0;exitDy=0;entryX=0.5;entryY=0;entryDx=0;entryDy=0;" edge="1" parent="1" source="oGheDNism3NP_6yb4b1f-3" target="oGheDNism3NP_6yb4b1f-34">
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="203" y="185" as="sourcePoint" />
+            <mxPoint x="233" y="185" as="targetPoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-22" value="yes" style="text;html=1;align=center;verticalAlign=middle;whiteSpace=wrap;rounded=0;fontSize=8;" vertex="1" parent="1">
+          <mxGeometry x="360" y="255" width="20" height="10" as="geometry" />
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-28" value="&lt;span style=&quot;font-size: 10px;&quot;&gt;Publicly available fix can be found in the Internet in mailing lists, kernel trees (linux-next, linus etc.)&lt;/span&gt;" style="rounded=0;whiteSpace=wrap;html=1;" vertex="1" parent="1">
+          <mxGeometry x="550" y="240" width="120" height="60" as="geometry" />
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-45" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;exitX=0.5;exitY=1;exitDx=0;exitDy=0;entryX=0.5;entryY=0;entryDx=0;entryDy=0;" edge="1" parent="1" source="oGheDNism3NP_6yb4b1f-34" target="oGheDNism3NP_6yb4b1f-14">
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-34" value="Bug is confirmed.&lt;div&gt;Fix is accepted.&lt;/div&gt;" style="rounded=0;whiteSpace=wrap;html=1;fillColor=#d5e8d4;strokeColor=#82b366;" vertex="1" parent="1">
+          <mxGeometry x="370" y="140" width="120" height="60" as="geometry" />
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-42" value="&lt;div&gt;&lt;span style=&quot;font-size: 10px;&quot;&gt;embargo period is up to 7 calendar days, with extension of 14 days.&lt;/span&gt;&lt;/div&gt;" style="rounded=0;whiteSpace=wrap;html=1;" vertex="1" parent="1">
```

Comment:
s/embargo/Embargo (please capitalize the phrases in the white blocks)

It is also unclear who is requesting the extension, and what is the maximum possible embargo period (is it 14 or 7+14 days?)

Reply:
I've clarified with the following:

> Embargo period is up to 7 calendar days, with the maximum additional 7 calendar days extension (14 days in total) which should be requested by the reporter with security@kernel.org and
linux-distros coordination.